code/go/0chain.net/smartcontract/utils/utils.go

```diff
@@ -0,0 +1,11 @@
+package utils
```

Comment:
These functions are blobber reward related, so it's better to put them in the package where blobber or reward structs are defined . `utils` is not a good package name actually, check the "Bad package names" section on https://go.dev/blog/package-names

Reply:
done